modules/es-cluster-health-is-red.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/troubleshooting-logging-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="es-cluster-health-is-red_{context}"]
```

Comment:
Per https://redhat-internal.slack.com/archives/C02321AKYTT/p1696947801776169?thread_ts=1696947268.418629&cid=C02321AKYTT - these need to be lowercase as part of the docs repo guidelines, but once this PR is merged the links shouldn't change again. Let's follow up in Slack to make sure the links in the code get updated properly once these docs are live, since apparently they're broken at the moment anyway.